packages/js/admin-layout/src/plugins/woo-footer-item/index.tsx

```diff
@@ -27,7 +29,10 @@ export const WC_FOOTER_SLOT_NAME = 'woocommerce_footer_item';
  * @param {Array}  param0.children - Node children.
  * @param {Array}  param0.order    - Node order.
  */
-export const WooFooterItem: React.FC< { order?: number } > & {
+export const WooFooterItem: React.FC< {
+	children?: React.ReactNode;
```

Comment:
Why did `children` was added here?

Reply:
In React 18, the `children` prop is no longer implicitly on `FC`. So, you have to explicitly define it.

https://www.newline.co/@kchan/what-happened-to-the-fc-types-implicit-children-prop-in-typesreact-v18--38576c59